app/services/api/charts/company.rb

```diff
@@ -0,0 +1,88 @@
+module Api
+  module Charts
+    class Company
+      SECTORS_LEVELS_DESC = {
+        '0' => 'Unaware of Climate Change as a Business Issue',
+        '1' => 'Acknowledging Climate Change as a Business Issue',
+        '2' => 'Building Capacity',
+        '3' => 'Integrating into Operational Decision Making',
+        '4' => 'Strategic Assessment'
+      }.freeze
+
+      def details_data(company)
```

Comment:
OK, so this isn't actually used by any chart. Maybe we could move it to regular views helpers.